### DISCUSSION TITLE
Using GUI

### DISCUSSION THREAD

**Discussion by BlackBaroness** [gorilla-devs/ferium#227] [Q&A] [answered]
Hi everyone, I know it's a CLI tool but I see there are GUI bin. I installed [ferium-gui-bin](https://aur.archlinux.org/packages/ferium-gui-bin) but I don't see anything about GUI. How can I enable it? If there are no GUI, why GUI bin exists?

**Comment by huantianad** [comment, marked as answer]
GUI refers to a "GUI file dialogue", based on the release notes.

**Reply by BlackBaroness** [reply]
> GUI refers to a "GUI file dialogue", based on the release notes.

Oh, yes, I didn't think to read the changelogs. But, oddly enough, I have not seen such functionality anywhere, although I used almost all the functions of the util.

**Reply by theRookieCoder** [reply]
The GUI file dialogue shows up whenever you need to select a directory, e.g. when you're creating a new profile and don't want to use the default `.minecraft` directory.

**Reply by BlackBaroness** [reply]
> The GUI file dialogue shows up whenever you need to select a directory, e.g. when you're creating a new profile and don't want to use the default `.minecraft` directory.

Strange, I made a profile with a custom folder, there was no window. I entered the path in the terminal.

**Reply by theRookieCoder** [reply]
~~Then it seems you didn't actually use the GUI binary?~~ edit: see #228

**Reply by BlackBaroness** [reply]
Does it need to be used in any particular way? I built the gui package from AUR and used `ferium` in terminal.

**Reply by theRookieCoder** [reply]
Huh interesting. I just checked and yeah the `4.2.0` release seems to be broken, it doesn't show the GUI file dialogue. There were some pretty significant changes to the build scripts in that update, so I might have made a mistake. Will check ASAP, thanks for bringing this to my attention!

**Reply by theRookieCoder** [reply]
I've created #228, it seems that I didn't update the `cfg`s when reworking libium's GUI feature flags.
